src/NUnitFramework/framework/Interfaces/TestOutput.cs

```diff
@@ -0,0 +1,82 @@
+// ***********************************************************************
+// Copyright (c) 2007 Charlie Poole
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+// 
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// ***********************************************************************
+
+using System;
+
+namespace NUnit.Framework.Interfaces
+{
+    /// <summary>
+    /// The TestOutput class holds a unit of output from 
+    /// a test to a specific output stream
+    /// </summary>
+	public class TestOutput
+	{
+        /// <summary>
+        /// Construct with text, ouput destination type and
+        /// the name of the test that produced the output.
+        /// </summary>
+        /// <param name="text">Text to be output</param>
+        /// <param name="stream">Destination of output</param>
+        /// <param name="testName">Name of teset that produced the output</param>
+		public TestOutput(string text, string stream, string testName)
+        {
+            Text = text;
+            Stream = stream;
```

Comment:
Is Stream a good name for this? It confused me until I looked at the type and the XMLDocs.